hbw/app/assets/javascripts/hbw_dependencies/noconflict.js.coffee

```diff
@@ -9,12 +9,12 @@
 #= require react
 #= require ./wrap_globals
 
-HBWDependencies = ['$', 'jQuery', 'moment', 'React']
+HBWDependencies = ['$', 'jQuery', 'moment', 'React', 'define']
```

Comment:
jQuery plugins which we use in HBW have algorithm for fetching current jQuery. For example select2.js:
```
function (factory) {
  if (typeof define === 'function' && define.amd) {
    // AMD. Register as an anonymous module.
    define(['jquery'], factory);
  } else if (typeof exports === 'object') {
    // Node/CommonJS
    factory(require('jquery'));
  } else {
    // Browser globals
    factory(jQuery);
  }
}
```
as you see, they firstly try to import from require.js.

It hurts a lot when integrated system has require.js under the hood (like CRM). In this case plugins for jQuery try to import jQuery from CRM's require.js.